subgraph/subgraph.template.yaml

```diff
@@ -1,7 +1,8 @@
 specVersion: 0.0.2
 schema:
-  file: ../schema.graphql
+  file: schema.graphql
 dataSources:
+  {{#liquity}}
```

Comment:
I think the `vault` section could be outside the `liquity` condition as it will be included in every subgraph right?

Reply:
yes I guess.